src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -943,24 +946,53 @@ private static SegueJobService getSegueJobService(final PropertiesLoader propert
                     new DeleteEventAdditionalBookingInformationOneYearJob()
             );
 
+            SegueScheduledJob eventReminderEmail = SegueScheduledJob.createCustomJob(
+                "eventReminderEmail",
+                "JavaJob",
+                "Send scheduled reminder emails to events",
+                "0 0 7 * * ?",
+                Maps.newHashMap(),
+                new EventReminderEmailJob()
+            );
+
+            SegueScheduledJob eventFeedbackEmail = SegueScheduledJob.createCustomJob(
+                "eventFeedbackEmail",
+                "JavaJob",
+                "Send scheduled feedback emails to events",
+                "0 0 20 * * ?",
+                Maps.newHashMap(),
+                new EventFeedbackEmailJob()
+            );
+
             SegueScheduledJob syncMailjetUsers = new SegueScheduledSyncMailjetUsersJob(
                     "syncMailjetUsersJob",
                     "JavaJob",
                     "Sync users to mailjet",
                     "0 0 0/4 ? * * *");
 
             List<SegueScheduledJob> configuredScheduledJobs = new ArrayList<>(Arrays.asList(PIISQLJob, cleanUpOldAnonymousUsers,
-                    cleanUpExpiredReservations, deleteEventAdditionalBookingInformation, deleteEventAdditionalBookingInformationOneYearJob));
+                    cleanUpExpiredReservations));
 
             if (mailjetKey != null && mailjetSecret != null) {
                 configuredScheduledJobs.add(syncMailjetUsers);
             }
 
+            if (Boolean.parseBoolean(eventPrePostEmails)) {
+                configuredScheduledJobs.add(eventReminderEmail);
+                configuredScheduledJobs.add(eventFeedbackEmail);
+            }
+
             segueJobService = new SegueJobService(configuredScheduledJobs, database);
 
             if (mailjetKey == null && mailjetSecret == null) {
                 segueJobService.removeScheduleJob(syncMailjetUsers);
             }
+
+            if (!Boolean.parseBoolean(eventPrePostEmails)) {
```

Comment:
Maybe you could do this parsing just once (i.e. make `eventPrePostEmails` itself the Boolean)?